pylib/gyp/common.py

```diff
@@ -424,38 +425,49 @@ def EnsureDirExists(path):
 
 def GetCrossCompilerPredefines():  # -> dict
     cmd = []
+
+    # shlex.split() will eat '\' in posix mode, but
+    # setting posix=False will preserve extra '"' cause CreateProcess fail on Windows
+    # this makes '\' in %CC_target% and %CFLAGS% work
+    def replace_sep(s):
+        return s.replace("\\", "/") if sys.platform == "win32" else s
```

Comment:
```suggestion
        return s.replace(os.sep, "/") if os.sep != "/" else s
```
https://docs.python.org/3/library/os.html#os.sep